streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedDemo.java

```diff
@@ -160,7 +167,8 @@ public PageViewByRegion apply(PageView view, UserProfile profile) {
                         return new KeyValue<>(viewRegion.region, viewRegion);
                     }
                 })
-                .countByKey(TimeWindows.of("GeoPageViewsWindow", 7 * 24 * 60 * 60 * 1000L).advanceBy(1000), Serdes.String())
+                .groupByKey(Serdes.String(), pageViewByRegionSerde)
```

Comment:
EDIT: now I realized it is because the previous `map` has changed the keys, and hence old serdes cannot be safely used any more, nvm.